controller/schema.go

```diff
@@ -436,6 +436,19 @@ $$ LANGUAGE plpgsql`,
 		$$ LANGUAGE plpgsql`,
 		`CREATE TRIGGER check_artifact_manifest AFTER INSERT ON artifacts FOR EACH ROW EXECUTE PROCEDURE check_artifact_manifest()`,
 	)
+	migrations.Add(27,
+		`CREATE TABLE sink_kinds (name text PRIMARY KEY)`,
+		`INSERT INTO sink_kinds (name) VALUES ('syslog')`,
+		`INSERT INTO event_types (name) VALUES ('sink'), ('sink_deletion')`,
```

Comment:
These need to be added to the [event schema](https://github.com/flynn/flynn/blob/master/schema/controller/event.json#L16-L36).